extra-resources/mods/Mythos/common.cardset

```diff
@@ -118,9 +145,38 @@ card('Healer') {
         }
     }
 }
-
+// https://upload.wikimedia.org/wikipedia/commons/e/ec/Ricci_Guangqi_2.jpg
+// CC BY-SA 3.0
+// https://commons.wikimedia.org/wiki/User:SteveMcCluskey
```

Comment:
_You must attribute the work **in the manner specified by the author or licensor** (but not in any way that suggests that they endorse you or your use of the work)._ (Creative Commons, emphasis mine)

On Wikimedia the attribution that should be used can be found clicking the _Use the file_ button. I'm not sure if this is really a violation of the CC BY, but either way I think it's a good gesture to the authors.

When I checked out [the Wikimedia page](https://commons.wikimedia.org/wiki/File:Ricci_Guangqi_2.jpg) for that image I saw another name listed in the attribution than the user name here:

> By Kircher, Athanasius, 1602-1680. [CC BY-SA 3.0 (http://creativecommons.org/licenses/by-sa/3.0)], via Wikimedia Commons